test/toy_object_test.rb

```diff
@@ -1,68 +1,87 @@
 require "test_helper"
 require "toy_object"
 
-describe ToyObject do
-  before do
-    @toy_object = ToyObject.new
-  end
+# Alternative: we could have created a new class, and delegated the missing
+# toy methods to Object's native implementation
+# class FakeToyObject
+#   def method_missing(selector, *args)
+#     selector = selector.to_s.sub(/^/, '')
```

Comment:
I woke up in a cold sweat with the horrible realisation I’d forgotten about `String#delete_prefix`:

```suggestion
#     selector = selector.to_s.delete_prefix("toy_")
```